src/main/java/net/fabricmc/loader/FabricLoader.java

```diff
@@ -249,6 +237,95 @@ private void setup() throws ModResolutionException {
 		}
 	}
 
+	/**
+	 * Prints the mod list according to the setting specified by
+	 * {@code -Dfabric_loader_modlist_style}. Valid values for this setting are:
+	 * <ul>
+	 *     <li>
+	 *         {@code verbose}. Prints each mod id, version, and path to the jar
+	 *         (including nested jars).
+	 *     </li>
+	 *     <li>
+	 *         {@code col[ncols=120]}. Prints mod id and version, arranged in a
+	 *         table to take up less space than {@code verbose} and be more
+	 *         readable than {@code short}.
+	 *         <br>
+	 *         The {@code ncols} setting determines the maximum width each line
+	 *         can take up. Using {@code -Dfabric_loader_modlist_style=col80}
+	 *         for example means that it will auto-wrap the printed text after
+	 *         80 characters. If {@code ncols} is left away, it will default to
+	 *         120.
+	 *     </li>
+	 *     <li>
+	 *         {@code short}. The OG. Prints a comma-seperated list of mod id
+	 *         and version.
+	 *     </li>
+	 * </ul>
+	 * The default value for {@code -Dfabric_loader_modlist_style} is 'col'.
+	 * @param candidateMap the mod candidate map to print
+	 */
+	private void printModList(Map<String, ModCandidate> candidateMap) {
+		String modText;
+		switch (candidateMap.size()) {
+			case 0:
+				modText = "Loading 0 mods";
+				break;
+			case 1:
+				modText = "Loading 1 mod: ";
+				break;
+			default:
+				modText = String.format("Loading %d mods: ", candidateMap.size());
+				break;
+		}
+
+		String style = System.getProperty("fabric_loader_modlist_style", "col");
+		int maxWidth = 120;
+
+		if (style.startsWith("col")) {
+			String width = style.substring(3);
+			if (!width.isEmpty()) {
+				try {
+					maxWidth = Integer.parseInt(width);
+				} catch (NumberFormatException e) {
+					LOGGER.error("Invalid max width for mod list output, ignoring: '{}'", width);
```

Comment:
Can we get an option to remove width limit?

Reply:
That max width is only for the column output, and it has to know the width to be able to make columns